test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -208,6 +212,28 @@ private CommandRunnerResult PackProjectOrSolution(string workingDirectory, strin
             return result;
         }
 
+        private CommandRunnerResult PackProjectOrSolutionWithRepoPackArtifacts(string workingDirectory, string file, string args, string nuspecOutputPath, bool validateSuccess)
+        {
+            var cliDirectory = Path.GetDirectoryName(TestDotnetCli);
+            var pathToSdkInCli = Path.Combine(
+                   Directory.GetDirectories(Path.Combine(cliDirectory, "sdk"))
+                       .First());
+            var pathOfNuGetBuildTasksPack = Path.Combine(pathToSdkInCli, "NuGet.Build.Tasks.Pack.dll");
+            var pathOfNuGetBuildTasksPackTragets = Path.Combine(pathToSdkInCli, "NuGet.Build.Tasks.Pack.targets");
+
+            var result = CommandRunner.Run(TestDotnetCli,
+                workingDirectory,
+                $"pack {file} {args} /p:ImportNuGetBuildTasksPackTargetsFromSdk=true /p:NuGetBuildTasksPackTargets={pathOfNuGetBuildTasksPackTragets} /p:NuGetPackTaskAssemblyFile={pathOfNuGetBuildTasksPack} /p:NuspecOutputPath={nuspecOutputPath}",
```

Comment:
nit: you don't need 
`/p:ImportNuGetBuildTasksPackTargetsFromSdk=true`, it's already set.

Reply:
A comment above - reverted MsbuildIntegrationTestFixture changes.